### PR TITLE
docs: add Dual Use Chemical Tracking System idea and example

### PR DESCRIPTION
**Pull request by ATREAY**

### CODE REVIEW

**Comment by ATREAY** [comment]
> @ATREAY you need to add your project details to two files [ideas.json](https://github.com/MakeContributions/ideahub/edit/main/data/ideas.json) or [projects.json](https://github.com/MakeContributions/ideahub/edit/main/data/projects.json)
> 
> Please read [Contributing Guidelines](https://github.com/MakeContributions/ideahub#contribution-guidelines)

I added my project details in projects.json file.
or is it like I need to add details in both the files.

**Comment by AryanP45** [comment]
> > @ATREAY you need to add your project details to two files [ideas.json](https://github.com/MakeContributions/ideahub/edit/main/data/ideas.json) or [projects.json](https://github.com/MakeContributions/ideahub/edit/main/data/projects.json)
> > Please read [Contributing Guidelines](https://github.com/MakeContributions/ideahub#contribution-guidelines)
> 
> I added my project details in projects.json file. or is it like I need to add details in both the files.

both files Please read [Contributing Guidelines](https://github.com/MakeContributions/ideahub#contribution-guidelines)

File: data/projects.json [hunk 1]
```diff
@@ -309,6 +309,16 @@
         "site": "https://movie-mania0.netlify.app/",
         "tags": "movies, api, react"
     },
+    {
+        "name": "Dual Use Chemical Tracking System",
+        "display": "DUCTS",
+        "description": "This software is designed for the NCB to track the supply chain of chemicals that can be illicitly used from the manufacturer to the end-user.",
```
Comment (AryanP45) [inline data/projects.json:315]:
```
"name": "DUCTS",
"display": "Dual Use Chemical Tracking System",
```

**Review by AryanP45 — CHANGES_REQUESTED** [review]
name should be Repository's name on github 
and display should me title

**Review by AryanP45 — APPROVED** [review]

**Comment by AryanP45** [comment]
LGTM

**Comment by ATREAY** [comment]
Any Updates??

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**